PyInstaller/building/imphook.py

```diff
@@ -123,6 +127,11 @@ def _cache_hook_dirs(self, hook_dirs):
                 # constructed by removing the "hook-" prefix and ".py" suffix.
                 module_name = os.path.basename(hook_filename)[5:-3]
 
+                if module_name in parsed_modules_hooks:
+                    continue
+                else:
```

Comment:
Please don't use `else` here. At a first glace it looks like the control folw would continue in the `if`-case, too.